Packs/MicrosoftADFS/README.md

```diff
@@ -3,11 +3,11 @@
 This pack includes XSIAM content.
```

Comment:
@nkanon please add instructions for winlogbeat as well.

Reply:
Added. Thanks Boss